client/extensions/wp-super-cache/components/navigation/index.jsx

```diff
@@ -65,12 +68,19 @@ const Navigation = ( { activeTab, site, translate } ) => {
 		} );
 	};
 
+	const pluginUrl = addSiteFragment( '/plugins/wp-super-cache', get( site, 'slug' ) );
 	return (
-		<SectionNav selectedText="Settings">
-			<SectionNavTabs>
-				{ renderTabItems( getTabs() ) }
-			</SectionNavTabs>
-		</SectionNav>
+		<div>
+			<HeaderCake backText={ translate( 'Plugin Overview' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **88** times:
`translate( 'Overview' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpsc-header-cake)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).